examples/cruise-control.py

```diff
@@ -347,7 +347,7 @@ def cruise_plot(sys, t, y, label=None, t_hill=None, vref=20, antiwindup=False,
 
 # Compute the equilibrium throttle setting for the desired speed (solve for x
 # and u given the gear, slope, and desired output velocity)
-X0, U0, Y0 = ct.find_eqpt(
+X0, U0, Y0 = ct.find_operating_point(
     cruise_pi, [vref[0], 0], [vref[0], gear[0], theta0[0]],
     y0=[0, vref[0]], iu=[1, 2], iy=[1], return_y=True)
```

Comment:
```suggestion
    y0=[0, vref[0]], iu=[1, 2], iy=[1], return_outputs=True)
```
Update the argument to avoid a `FutureWarning` being raised in the example.

Reply:
Similarly, `return_y=True` in [examples/cruise.ipynb](https://github.com/python-control/python-control/blob/cc0020a990bcb96ade16e338f5b152884fa4fe16/examples/cruise.ipynb) should be updated, too.